platform/security/filter/security-filter-authorization/src/main/java/org/codice/ddf/security/filter/authorization/AuthorizationFilter.java

```diff
@@ -83,9 +83,9 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 
             boolean permitted = true;
             if (shouldActionAuthorize) {
```

Comment:
Removed shouldActionAuthorize block. Also removed metatype for this since it isn't used anywhere else.